web/src/App.tsx

```diff
@@ -1,11 +1,11 @@
 import React from "react";
 
 import { Providers } from "@features/Providers/Providers";
-import { Routes } from "@routing/Routes";
+import { RouterRoutes } from "@routing/Routes";
 
 const App: React.VFC = () => (
   <Providers>
-    <Routes />
+    <RouterRoutes />
```

Comment:
It's pretty obvious, that `Routes` are child of a `Router`. Can we rename it?

PS What do you think about taking `Router` out of `Providers` component and move it somewhere to `@routing/...`? IMO routing logic should stay in one place, so developer is able to review routing logic easily.